shared/src/main/scala/coop/rchain/lmdb/LMDBStore.scala

```diff
@@ -28,14 +34,18 @@ final case class LMDBStore[F[_]: Sync](env: Env[ByteBuffer], dbi: Dbi[ByteBuffer
     }
 
   def withReadTxnF[R](f: Txn[ByteBuffer] => R): F[R] =
-    Sync[F].delay {
-      withTxn(env.txnRead)(f)
-    }
+    Sync[F]
+      .delay {
+        withTxn(env.txnRead)(f)
+      }
+      .timer("lmdb.read")
```

Comment:
we need to educate about this metric and we're not allowed to just draw out conclusions